services/dashboard/views/cluster/master.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed under the Apache License, Version 2.0
+ * See accompanying LICENSE file.
+ */
+'use strict';
+
+angular.module('dashboard.cluster', [])
+
+  .config(['$routeProvider', function ($routeProvider) {
+    $routeProvider
+      .when('/cluster', {label: 'Cluster', redirectTo: '/cluster/master'})
+      .when('/cluster/master', {
+        label: 'Master',
+        templateUrl: 'views/cluster/master.html',
+        controller: 'MasterCtrl'
+      });
+  }])
+
+  .controller('MasterCtrl', ['$scope', 'restapi', function ($scope, restapi) {
+    $scope.statusClass = function (status, good, bad) {
+      return status === 'synced' ? good : bad;
+    };
+
+    restapi.subscribe('/master', $scope,
```

Comment:
Add reference here to the data type MasterData. Something like:
//TODO datatype: MasterData. Try and do serde using scalajs See #458